packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -32,6 +32,7 @@ dependencies:
   jaguar_jwt: ^3.0.0 # This package is used for the local server (if not used, can be removed)
   json_annotation: ^4.7.0
   jwt_decoder: ^2.0.1
+  notification_permissions: ^0.6.1
```

Comment:
I'm wondering won't it be better to use: https://pub.dev/packages/permission_handler for the permissions? It appears to require some initial setup, but is well supported and can also be used in the future if other permissions need to be included in the app.

Reply:
Swapped notification_permissions with permission_handler 👍